### DISCUSSION TITLE
workspaceFolder setting doesn't work

### DISCUSSION THREAD

**Discussion by mtsmfm** [community/community#25449] [Codespaces]
<div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<pre><code class="lang-json">{
  "dockerComposeFile": "./docker-compose.yml",
  "service": "app",
  "workspaceFolder": "/app",
  "containerUser": "app"
}
</code></pre>
<p>When I connect to the codespace, I’m on <code>~/workspace/gh-codespaces-example</code>, not <code>/app</code></p>
<pre><code class="lang-auto">app@82bf4527040e:~/workspace/gh-codespaces-example$ pwd
/home/app/workspace/gh-codespaces-example
</code></pre>

**Comment by jkeech** [comment, marked as answer]
<p>Hi <a class="mention" href="/mtsmfm">@mtsmfm</a>, the <code>workspaceFolder</code>  property is not supported in Codespaces yet, but we are working on adding support for that in the coming weeks. Right now it always clones the repo to <code>/home/{remoteUser}/workspace/{repoName}</code>.</p>

**Reply by davetapley** [reply]
@jkeech do you know if this was ever added? See: https://github.com/community/community/discussions/11923#discussioncomment-4347853

**Comment by mtsmfm** [comment]
<p><a class="mention" href="/jkeech">@jkeech</a> Thank you for your response.</p>
<p>I sent a PR to fix the doc I read.<br>
</p><aside class="onebox githubpullrequest">
  <header class="source">
      <a href="https://github.com/github/docs/pull/578" target="_blank" rel="noopener nofollow ugc">github.com/github/docs</a>
  </header>
  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Pull Request">
    <svg width="60" height="60" class="github-icon" viewbox="0 0 12 16" aria-hidden="true"><path d="M11 11.28V5c-.03-.78-.34-1.47-.94-2.06C9.46 2.35 8.78 2.03 8 2H7V0L4 3l3 3V4h1c.27.02.48.11.69.31.21.2.3.42.31.69v6.28A1.993 1.993 0 0 0 10 15a1.993 1.993 0 0 0 1-3.72zm-1 2.92c-.66 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2zM4 3c0-1.11-.89-2-2-2a1.993 1.993 0 0 0-1 3.72v6.56A1.993 1.993 0 0 0 2 15a1.993 1.993 0 0 0 1-3.72V4.72c.59-.34 1-.98 1-1.72zm-.8 10c0 .66-.55 1.2-1.2 1.2-.65 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2zM2 4.2C1.34 4.2.8 3.65.8 3c0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/github/docs/pull/578" target="_blank" rel="noopener nofollow ugc">workspaceFolder is not supported yet</a>
    </h4>

    <div class="branches">
      <code>github:main</code> ← <code>mtsmfm:patch-1</code>
    </div>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2020-10-16" data-time="02:49:34" data-timezone="UTC">02:49AM - 16 Oct 20 UTC</span>
      </div>

      <div class="user">
        <a href="https://github.com/mtsmfm" target="_blank" rel="noopener nofollow ugc">
          <img alt="mtsmfm" src="https://user-images.githubusercontent.com/1796864/181091173-e5b82dbf-c2ad-4191-bb4f-6a149c3c5638.jpeg" class="onebox-avatar-inline" width="20" height="20">
          mtsmfm
        </a>
      </div>

      <div class="lines" title="1 commits changed 1 files with 1 additions and 1 deletions">
        <a href="https://github.com/github/docs/pull/578/files" target="_blank" rel="noopener nofollow ugc">
          <span class="added">+1</span>
          <span class="removed">-1</span>
        </a>
      </div>
    </div>

  </div>
</div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
